.github/workflows/windows_ci.yml

```diff
@@ -0,0 +1,104 @@
+# Used the following as reference:
```

Comment:
I think you can drop it for now. As long as we can confirm it works on linux (and mac too I guess), it's fine for now. Windows CI is always a nightmare. If you want to put it into a follow-up PR, it'll be simpler since the CI will actually test it.

Reply:
Alright, I will drop this.